src/pages/privacy.md

```diff
@@ -39,7 +39,8 @@ services. This includes the following:
 - Importing or exporting data to Google Sheets or Google Drive;
 - Importing or exporting data to a SQL database;
 - Importing a file retrieved from a URL;
-- Uploading data to Wikibase instances.
+- Uploading data to Wikibase instances;
+- Enabling the OpenREfine release and events notifications (starting OpenRefine 3.8).
```

Comment:
I wouldn't make this part of the same category. So far this list is about operations which transmit project data to/from external services. Release notifications transmit only IP address and browser metadata, which feels like a very different type of data transfer.

Also, I wouldn't write `starting OpenRefine 3.8` because those requests have been happening for as long as I can remember (to GitHub.com, enabled by default). OpenRefine 3.8 will make that opt-in instead.